app/views/application/_nav.html.erb

```diff
@@ -7,6 +7,25 @@
       <strong><%= link_to 'Vesta', root_path %></strong>
     </div>
     <div id="responsive-menu">
+      <div class="top-bar-left">
```

Comment:
There needs to be logic here around what is available to different users / in different states. For example, we should probably only show the `Log In` link when a user is not logged in. Additionally, all of these links feel like admin-only links so they should likewise be restricted to admin users. Does that make sense?